Tests/Pcap++Test/Tests/LoggerTests.cpp

```diff
@@ -71,28 +71,28 @@ std::string* LogPrinter::lastFilenameSeen = nullptr;
 std::string* LogPrinter::lastMethodSeen = nullptr;
 int LogPrinter::lastLineSeen = 99999;
 
-
 class MultiThreadLogCounter
 {
-	public:
-		static const int ThreadCount = 5;
-		static int logMessageThreadCount[ThreadCount];
-		static void logPrinter(pcpp::Logger::LogLevel logLevel, const std::string& logMessage, const std::string& fileName, const std::string& method, const int line)
-		{
-			int threadId = logMessage[logMessage.length() - 1] - '0';
-			MultiThreadLogCounter::logMessageThreadCount[threadId]++;
-		}
+public:
+	static const int ThreadCount = 5;
+	static int logMessageThreadCount[ThreadCount];
+	static void logPrinter(pcpp::Logger::LogLevel logLevel, const std::string& logMessage, const std::string& fileName,
+	                       const std::string& method, const int line)
+	{
+		int threadId = logMessage[logMessage.length() - 1] - '0';
+		MultiThreadLogCounter::logMessageThreadCount[threadId]++;
+	}
 };
 
-int MultiThreadLogCounter::logMessageThreadCount[MultiThreadLogCounter::ThreadCount] = {0, 0, 0, 0, 0};
-
+int MultiThreadLogCounter::logMessageThreadCount[MultiThreadLogCounter::ThreadCount] = { 0, 0, 0, 0, 0 };
 
+// clang-format off
```

Comment:
don't know why, but seems the formatting of this part will break the tests.

Reply:
What the diference between before and after clang-format ?